apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1145,14 +1145,17 @@ def _write_dataframe(
     axis_mapping: AxisIDMapping,
 ) -> DataFrame:
     """
-    The id_column_name is for disambiguating rows in append mode;
-    it may or may not be an index name in the input AnnData obs/var.
+    Convert and save a pd.DataFrame as a SOMA DataFrame.
```

Comment:
```suggestion
    Convert and save a ``pd.DataFrame`` as a SOMA ``DataFrame``.
```

And likewise throughout.

I may not have remember this on every PR review -- my apologies -- I don't always remember to flag it. But we _should_ always. Issue being our status-quo docsite is here

https://tiledbsoma.readthedocs.io/en/latest/

and we need RTD-style markup including double backticks

Reply:
BTW if we ever "miss any" I'm happy for one of us to put up an RTD-docstring-formats-only after-PR, no questions asked :)